app/views/pages/terms_and_conditions.html.erb

```diff
@@ -161,18 +161,29 @@
         <h3 class="govuk-heading-s">Accepting applications through the Get school experience service</h3>
             <p>
                 By accepting school experience applications through the Service, you agree that:
-            <p>
-            <p>
-                Once you receive an application from a school experience candidate, it's your responsibility to adhere to General Data Protection Regulation (GDPR) requirements.
             </p>
                 <ul class="govuk-list govuk-list--bullet">
+                    <li>Once you receive an application from a school experience candidate, it's your responsibility to adhere to General Data Protection Regulation (GDPR) requirements.</li>
                     <li>you're responsible for destroying any digital, downloaded, printed or saved copies of a school experience request</li>
                     <li>you must not share requests received through the Service with anyone other than those involved in the school experience process from the school, trust, board or local authority</li>
                     <li>you must ensure that appropriate measures are in place to guarantee the protection of personal data in applications</li>
                 </ul>
             <p>
                 As a school, it's your responsibility to decide if a school experience candidate complies with your DBS and safeguarding policies.
             </p>
+        <h3 class="govuk-heading-s">Data storage, retention and destruction</h3>
+            <p>
+                You must store shared data securely and have appropriate safeguards in place to protect data against unlawful or unauthorised processing. Personal data must be kept only as long as needed to carry out activities relating to administering school experience.
+            </p>
+            <p>
+                You must set an appropriate time limit, in line with the General Data Protection Regulation, for retaining data before erasure or review.
+            </p>
+            <p>
+                You should destroy the data when it is no longer needed, or when the retention schedule has expired.
+            </p>
+            <p>
+                You should follow the NCSC guidance for secure sanitisation.
```

Comment:
```suggestion
                You should follow the National Cyber Security Centre (NCSC) guidance for secure sanitisation.
```
think we need to put this in full as it is not defined elsewhere on the page

we _could_ include a link to the guidance (do Manage?)

Reply:
Manage do not, I propose we leave it for now.